code/modules/unit_tests/unit_test.dm

```diff
@@ -19,20 +19,33 @@ GLOBAL_VAR(test_log)
 	//Bit of metadata for the future maybe
 	var/list/procs_tested
 
-	//usable vars
+	/// The bottom left turf of the testing zone
 	var/turf/run_loc_bottom_left
+
+	/// The top right turf of the testing zone
 	var/turf/run_loc_top_right
 
+	/// The type of turf to allocate for the testing zone
+	var/test_turf_type = /turf/open/floor/plasteel
+
 	//internal shit
 	var/focus = FALSE
 	var/succeeded = TRUE
 	var/list/allocated
 	var/list/fail_reasons
 
+	var/static/datum/turf_reservation/turf_reservation
```

Comment:
This was added initially because I thought the reason the test was failing was because of a bug where the N2O chem sometimes releases as a gas. I thought it was filling the parts of the testing environment that *aren't* space with N2O, and thus was making the humans fall asleep. It's only still in here because it's something we should be using anyway.